n2v/utils/n2v_utils.py

```diff
@@ -73,7 +73,6 @@ def local_gaussian(patch, coords, dims):
         return vals
     return local_gaussian
 
-
```

Comment:
According to [PEP8 Python formatting guide lines](https://www.python.org/dev/peps/pep-0008/#blank-lines) top-level functions should be surrounded by two blank lines.